### DISCUSSION TITLE
[Feature] Unified view of User and Partner media in one timeline

### DISCUSSION THREAD

**Discussion by imthefrizzlefry** [immich-app/immich#5526] [Feature Request]
### The feature

As a user with a linked partner
I want the ability to view partner media in my timeline
So that I can have all our photos in one view

In the photo app that shall not be named, a user can choose to automatically save partner photos to their photo collection.  The full feature in the other app is linked to other more advanced filtering options, but this feature would be an all or nothing request.  The expected behavior would indicate that if I select to copy partner photos to my timeline, then at least all future photos will get copied to my collection.  This would double memory consumption, but provide a minimal feature parity.

- Stretch goal 1: Selecting to copy partner photos would trigger a job to copy existing photos.
- Stretch goal 2: When triggering a batch job to copy existing photos, the user could select a date for the oldest photo to copy.  For example, only copy photos after January 1st 2020.
- Stretch goal 3:  Reduce duplication of photos by storing all photos in a common folder for all users, then have a permissions layer to determine what photos each user has access to.  This would require logic to track which versions of each file each user has access to.  Also, it would require soft deletion, where a file could be deleted for one user, but still visible to other users.

### Platform

- [X] Server
- [ ] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
You can enable unified timeline in the partner sharing settings.
